### DISCUSSION TITLE
achieved milestones history

### DISCUSSION THREAD

**Discussion by yigicool** [KiARC/Sobriety#101] [Ideas]
Hello here,
I wonder if it'd be motivating or demotivating for people to have a reached milestones history.

For me it'd probably be helpful to think, "did it once, can do it again".

Still I rather ask for opinion before I ask for it as a new feature in this super app.

Would b helpful to write for or against with a small comment.
Then I'll come here in a few days to check the results or maybe @KiARC can decide if helpful beforehand.

**Comment by seaglade** [comment]
Personally, for. I think this is a great idea. But of course I'll let other people comment on it too so I can get a better sense of what people think.
